tests/files/tz-identifier.parseformat

```diff
@@ -2,4 +2,4 @@ TS: 0 | -99999--99999--99999 01:00:03 0.123450 Europe/Amsterdam | H:i:s.u e | 01
 TS: 0 | -99999--99999--99999 01:00:03 0.123450 America/Indiana/Knox | H:i:s.u e | 01:00:03.12345 America/Indiana/Knox
 TS: 0 | 2005-07-14 22:30:41 America/Los_Angeles | Y-m-d H:i:s e | 2005-07-14 22:30:41 America/Los_Angeles
 TS: 0 | 2005-07-14 22:30:41 America/Los_Angeles | Y-m-d?H:i:s?e | 2005-07-14	22:30:41	America/Los_Angeles
-TS: 0 | 2020-02-25 10:21:25 CET 03600  0Y   0M   0D /   0H   0M   0S / 2.1 | D-M-d-H.i.s-T-Y | Tue-Feb-25-10.21.25-CET-2020
\ No newline at end of file
+TS: 0 | 2020-02-25 10:21:25 CET  0Y   0M   0D /   0H   0M   0S / 2.1 | D-M-d-H.i.s-T-Y | Tue-Feb-25-10.21.25-CET-2020
```

Comment:
I am not sure which form is more proper here
The latter is returned if i check for `tp->type == TIMELIB_ZONETYPE_ABBR`

Reply:
Ok so it seems `CET 03600` was printed because the code thought the time was of type `TIMELIB_ZONETYPE_OFFSET`, which is not correct.

I'd say the new format is more proper, but it looks like in both cases `have_relative` has been set to 1 somehow, which turned this into a relative time.
This seems wrong, i'd need to investigate into why

Reply:
After investigating, it seems the relative format is triggered by 'D' 
```c
{'D', TIMELIB_FORMAT_TEXTUAL_DAY_3_LETTER},
```
so i'd say it's behaving correctly

Reply:
Please make sure to have the last line end with a `\n` though. The new test line doesn't belong in this file though, as it's for testing **identifiers**. Please add a new test file `tests/files/tz-abbreviation.parseformat` and add a **set** of tests there. Please use `:` as time separator for the results to be more easily parsable by people.

Please also add a "c test" into `tests/c/parse_date_from_format_test.cpp`.